srcpkgs/waylock/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'waylock'
 pkgname=waylock
-version=0.3.1
+version=0.3.2
 revision=1
 build_style=cargo
-makedepends="cairo-devel pam-devel"
+makedepends="wayland-devel cairo-devel pam-devel"
 short_desc="Simple screenlocker for wayland compositors"
-maintainer="travankor <travankor@tuta.io>"
+maintainer="Isaac Freund <ifreund@ifreund.xyz>"
 license="MIT"
 homepage="https://github.com/ifreund/waylock"
 distfiles="https://github.com/ifreund/waylock/archive/v${version}.tar.gz"
```

Comment:
Want to do the `${homepage}/archive...` tricK for distfiles?

Reply:
Good call, done